<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2123</width>
    <height>883</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblTitulo">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Proveedores</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAnadirProveedor">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>440</y>
      <width>331</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>AÃ±adir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnModificarProveedor">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Modificar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEliminarProveedor">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>440</y>
      <width>351</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnVolver">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaProveedores">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>511</width>
      <height>192</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>4</number>
    </property>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QWidget" name="imagen" native="true">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>60</y>
      <width>411</width>
      <height>651</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2123</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
